hub/README.md

```diff
@@ -1,3 +1,5 @@
+The rest of this will probably be refactored
+
 # Running the Gaia Hub
 
```

Comment:
This README actually says further down:

```
The Gaia hub currently supports the following drivers:

```
'aws' == Amazon S3
'azure' == Azure Blob Storage
'disk' == Local disk (you must set up static web-hosting to point at this driver)
'google-cloud' === Google Cloud Storage
```

Set the driver you wish to use in your `config.json` file with the `driver` parameter. Many drivers additionally accept the `bucket` parameter, which controls the bucket name that files should be written to.
```

Do we actually want to publish this in our README this way without a caveat to testing and so forth?

Reply:
that line was added here:
https://github.com/blockstack/gaia/commit/f912de9364aac4cce67722718e9656d98bfe8565#diff-f544082c9f728f2d36ca08fe1a5843f3

@GInxh , not sure of the reasoning behind this addition.